docs/cse/administration/create-use-network-blocks.md

```diff
@@ -90,11 +90,13 @@ The table below defines the fields you can import for a Network Block.
 
 Here is an example of a file in which all fields are supplied:
 
-`address_block,label,internal,suppresses_signals 10.0.5.0/24,”Internal Block”,true,false`
+```address_block,label,internal,suppresses_signals
+10.0.5.0/24,”Internal Block”,true,false```
```

Comment:
To do code blocks, the backticks need to be on their own lines (sorry I didn't mention that before)
<img width="361" alt="Screenshot 2023-04-21 at 9 53 53 PM" src="https://user-images.githubusercontent.com/56411016/233763470-c26f3a83-718d-48e7-8965-862ac64a6cf3.png">